### PR TITLE
v5.8 Changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/administration/changelog.md [hunk 1]
```diff
@@ -4,6 +4,105 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
 
 Also see [changelog in progress](http://bit.ly/2nK3cVf) for the next release.
 
+## Release v5.8
+
+Release Date 16/02/2019
```
Comment (esethna) [inline source/administration/changelog.md]: check date format on past releases

File: source/administration/changelog.md [hunk 2]
```diff
@@ -4,6 +4,105 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+#### Notifications
+ - Added a channel notification setting to disable at-channel mentions.
+ 
+#### Administration
+ - Added the ability to search users by role in **System Console** > **Users**.
+ - Added a CLI command to modify an outgoing webhook.
+ - Added a CLI command to restore a team.
+
+#### Performance
+ - Made posting idempotent.
```
Comment (esethna) [inline source/administration/changelog.md]: What is the benefit? We should highlight benefits here

File: source/administration/changelog.md [hunk 3]
```diff
@@ -4,6 +4,105 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+
+#### Slash Commands
+ - Added support for sending a message to a different channel than where the slash command was issued from.
```
Comment (esethna) [inline source/administration/changelog.md]: Does this apply to more than just GMs/DMs? What is the command?

File: source/administration/changelog.md [hunk 4]
```diff
@@ -4,6 +4,105 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+ - Improved error handling of slash commands.
+ - Added an option to send a message when an invalid slash command is entered on the right-hand side.
```
Comment (esethna) [inline source/administration/changelog.md]: Added an option to send a message beginning with a "/" from the right-hand side.

File: source/administration/changelog.md [hunk 5]
```diff
@@ -4,6 +4,105 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+ - Added the ability to search users by role in **System Console** > **Users**.
+ - Added a CLI command to modify an outgoing webhook.
+ - Added a CLI command to restore a team.
+
+#### Performance
+ - Made posting idempotent.
+
+#### Slash Commands
+ - Added support for sending a message to a different channel than where the slash command was issued from.
+ - Improved error handling of slash commands.
```
Comment (esethna) [inline source/administration/changelog.md]: What's the benefit?

File: source/administration/changelog.md [hunk 6]
```diff
@@ -4,6 +4,105 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+ - Added an option to send a message when an invalid slash command is entered on the right-hand side.
+
+#### Plugins
+ - Added server support for updating a plugin instead of having to remove and install them as two separate actions.
+ 
+#### Attachments
+ - Optimized file attachment memory usage where possible.
+ 
+#### Integrations 
+ - Enabled "true"/"false" for bool values in webhook payload.
```
Comment (esethna) [inline source/administration/changelog.md]: Not sure I understand this one fully. Can we clarify

File: source/administration/changelog.md [hunk 7]
```diff
@@ -4,6 +4,105 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+Multiple setting options were added to `config.json`. Below is a list of the additions and their default values on install. The settings can be modified in `config.json`, or the System Console when available.
+ 
+#### Changes to Team Edition and Enterprise Edition:
+ 
+ - Under ``"ServiceSettings"`` in ``config.json``:
+    - Added ``"ExperimentalLdapGroupSync": false``, ``"GroupFilter": ""``, ``"GroupDisplayNameAttribute": ""``, ``"GroupIdAttribute": ""`` and ``"ExperimentalStrictCSRFEnforcement": false``, to add the ability to configure group display name and unique identifier.
+ - Under ``"ImageProxySettings":`` in ``config.json``:
+    - Added ``"Enable": true,``, ``"ImageProxyType": "local",``, ``"RemoteImageProxyURL": "",`` and ``"RemoteImageProxyOptions": ""``, to allow integrating image proxy into the server and switching it on by default.
+ - Under ``"ExperimentalSettings":`` in ``config.json``:
+    - Added ``"LinkMetadataTimeoutMilliseconds": 5000``, to
```
Comment (esethna) [inline source/administration/changelog.md]: What about the settings for experimental post metadata?

"DisablePostMetadata": false, was added under experimental settings

File: source/administration/changelog.md [hunk 8]
```diff
@@ -4,6 +4,105 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+
+### API Changes
+
+#### RESTful API v4 Changes
+ - Added ``SearchTeams`` to plugin API to add the ability to search teams.
+ - Added ``GetTeamStats`` to plugin API to add the ability to get team statistics.
+ - Added a new API endpoint to get the bulk reactions for posts.
```
Comment (esethna) [inline source/administration/changelog.md]: What is the API endpoint?

File: source/administration/changelog.md [hunk 9]
```diff
@@ -4,6 +4,102 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+### Compatibility
+
+### config.json
+
+Multiple setting options were added to `config.json`. Below is a list of the additions and their default values on install. The settings can be modified in `config.json`, or the System Console when available.
+ 
+#### Changes to Team Edition and Enterprise Edition:
+ 
+ - Under ``"ServiceSettings"`` in ``config.json``:
+    - Added ``"ExperimentalLdapGroupSync": false``, ``"GroupFilter": ""``, ``"GroupDisplayNameAttribute": ""``, ``"GroupIdAttribute": ""`` and ``"ExperimentalStrictCSRFEnforcement": false``, to add the ability to configure group display name and unique identifier.
```
Comment (lieut-data) [inline source/administration/changelog.md]: Looks like `ExperimentalStrictCSRFEnforcement` requires a description here?

**Pull request merged**